src/Athena/Views/Student/ConfigureCompletedCourses.cshtml

```diff
@@ -22,30 +22,17 @@
                         <a class="modal-trigger color-inherit" href="#add-completed-courses-dialog"><i class="material-icons small">playlist_add</i></a> 
                     </div>
                 </div>
-                <div class="row" id="completed-courses">
+                <div class="row" id="completed-courses" style="overflow-y: scroll; max-height :25rem">
                     <div class="col s12">
-                        <div class="progress"><div class="indeterminate"></div></div>
+                        <div class="progress"><div class="indeterminate"></div></div> 
                     </div>        
                 </div>
-                <div class="row">
-                    <div class=" col s12">
-                        <ul class="pagination pagination-dark center">
-                            <li class="disabled"><a href="#!"><i class="material-icons">chevron_left</i></a></li>
-                            <li class="active"><a href="#!">1</a></li>
-                            <li class="waves-effect"><a href="#!">2</a></li>
-                            <li class="waves-effect"><a href="#!">3</a></li>
-                            <li class="waves-effect"><a href="#!">4</a></li>
-                            <li class="waves-effect"><a href="#!">5</a></li>
-                            <li class="waves-effect"><a href="#!"><i class="material-icons">chevron_right</i></a></li>
-                        </ul>
-                    </div>
-                </div>
             </div>
         </div>
     </div>
 </div>
 <div id="add-completed-courses-dialog" class="modal modal-fixed-footer">
-    <div class="modal-content">
+    <div class="modal-content scrolling">
```

Comment:
Is there already a style rule for `.scrolling`? I don't see any stylesheet changes in this diff